packages/tracing/src/transaction.ts

```diff
@@ -122,6 +121,8 @@ export class Transaction extends SpanClass implements TransactionInterface {
       }).endTimestamp;
     }
 
+    const metadata = this.metadata;
+
```

Comment:
Are we doing this (+ the references to `metadata` instead of `this.metadata` below) to save bundle size?
(No action required)